src/main/java/org/folio/marc4ld/service/marc2ld/mapper/mapper/OtherVersionMapper.java

```diff
@@ -0,0 +1,114 @@
+package org.folio.marc4ld.service.marc2ld.mapper.mapper;
+
+import static org.folio.ld.dictionary.PredicateDictionary.INSTANTIATES;
+import static org.folio.ld.dictionary.PredicateDictionary.MAP;
+import static org.folio.ld.dictionary.PredicateDictionary.OTHER_VERSION;
+import static org.folio.ld.dictionary.PropertyDictionary.LABEL;
+import static org.folio.ld.dictionary.PropertyDictionary.NAME;
+import static org.folio.ld.dictionary.ResourceTypeDictionary.IDENTIFIER;
+import static org.folio.ld.dictionary.ResourceTypeDictionary.ID_LCCN;
+import static org.folio.ld.dictionary.ResourceTypeDictionary.ID_UNKNOWN;
+import static org.folio.marc4ld.util.Constants.S;
+import static org.folio.marc4ld.util.Constants.T;
+import static org.folio.marc4ld.util.Constants.TAG_776;
+import static org.folio.marc4ld.util.MarcUtil.getSubfieldValue;
+
+import java.util.List;
+import java.util.Set;
+import lombok.RequiredArgsConstructor;
+import org.folio.ld.dictionary.PredicateDictionary;
+import org.folio.ld.dictionary.ResourceTypeDictionary;
+import org.folio.ld.dictionary.model.Resource;
+import org.folio.ld.dictionary.model.ResourceEdge;
+import org.folio.ld.fingerprint.service.FingerprintHashService;
+import org.folio.marc4ld.dto.MarcData;
+import org.folio.marc4ld.service.label.LabelService;
+import org.folio.marc4ld.service.marc2ld.mapper.Marc2ldMapper;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+public class OtherVersionMapper implements Marc2ldMapper {
+
+  private static final Set<ResourceTypeDictionary> IDENTIFIER_TYPE = Set.of(IDENTIFIER);
+  private static final String DLC_PREFIX = "(DLC)";
+
+  private final MapperHelper mapperHelper;
+  private final LabelService labelService;
+  private final FingerprintHashService hashService;
+
+  @Override
+  public String getTag() {
```

Comment:
I think this line will not compile. We recently changed the method in Marc2ldMapper to `getTags()`

Reply:
My bad. I performed ```mvn clean install``` before rebasing to the new master branch and forgot to do it after the rebase

Reply:
Fixed